libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -531,8 +531,8 @@ void AC_PosControl::set_pos_target(const Vector3f& position)
 {
     _pos_target = position;
 
-    _flags.use_desvel_ff_z = false;
-    _vel_desired.z = 0.0f;
+    //_flags.use_desvel_ff_z = false;
```

Comment:
Please change this comit message to:
AC_AttitudeControl: Added altitude velocity feedforward for posvel controller

Reply:
done

Reply:
Why did you do this?

Reply:
Because when you use the combined position-velocity controller, the velocity commands didn’t go through.
The drone just kept hanging still in the air, regardless of any x, y or z velocity input.
This way the velocity input is directly fed forward.

Reply:
Did you consider what this would do to all the other modes that used this function?